src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -144,9 +144,17 @@
 
     <!-- If VersionsFiles were passed on the command-line, convert them to an item group -->
     <ItemGroup>
-      <_VersionsFiles Include="$(VersionsFiles)" />
+      <_VersionsFiles Include="$(VersionsFiles)" />    
+      <SupportsDefinitionFileGroup Condition="'$(SupportsDefinitionFile)' == ''" Include="$(MSBuildThisFileDirectory)../packages/Microsoft.DotNet.BuildTools.TestSuite/**/runtime.json" />
```

Comment:
Yes I would prefer we don't do wildcard searching under the packages folder. Do we think TestRuntimeProjectJson file is going to be the right file for this case? It has a different set of supports clauses.